include/quintain.h

```diff
@@ -35,6 +35,17 @@ extern "C" {
 /* flags for workload operations */
 #define QTN_WORK_USE_SERVER_POOLSET 1
 
+/* op flags */
+typedef enum {
+    QTN_NOOP        = 0,
+    QTN_BULK_PULL   = 1,
```

Comment:
I've never tried this in an enumeration before, but if it works I have a slight preference for setting the values with bit shifts, like `1<<0` `1<<1` `1<<2` etc. so that it's easier to check at a glance.  Maybe comment here too to explain that they are powers of 2 so that they can be combined.